pkg/security/rules/engine.go

```diff
@@ -183,6 +183,24 @@ func (e *RuleEngine) Start(ctx context.Context, reloadChan <-chan struct{}, wg *
 			}
 		}
 	}()
+
+	// Sending an heartbeat event every minute
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+
+		heartbeatTicker := time.NewTicker(1 * time.Minute)
```

Comment:
The frequency at which the heartbeat event is generated 1 minutes, let me know if you disagree, that might be too much